src/model_config_tests/models/accessom3.py

```diff
@@ -0,0 +1,90 @@
+"""Specific Access-OM3 Model setup and post-processing"""
+
+import re
+from collections import defaultdict
+from pathlib import Path
+from typing import Any
+
+from payu.models.cesm_cmeps import Runconfig
```

Comment:
Sorry to bring this up so late @jo-basevi, but maybe we don't want to do it this way?

If we need to reuse `Runconfig` then should it be in its own package?

Does something like `om3-utils` also need to reuse this or similar manipulation routines?

Reply:
`om3-utils` doesn't reuse this, but it does have similar functionality. We could use `om3-utils` instead, but is that any better?

Reply:
Maybe in future when `om3-utils` is packaged, could use reuse `om3-utils` instead. 

When running repro tests on gadi, if payu is a dependency it needs to be installed inside the test environment. Otherwise if `payu` is not a dependency, `payu` can be obtained from `g/data/vk83/apps/payu` modules, as it just needs the `payu run` command.

So I think it's fine to keep payu as a dependency for now. A benefit is that it keeps the test environment more isolated. But if the number of files of the test environments becomes a problem, then removing payu as a dependency could maybe help.